cli/tests/it/cmd.rs

```diff
@@ -1341,3 +1341,23 @@ forgetest_init!(can_install_missing_deps_build, |prj: TestProject, mut cmd: Test
     assert!(output.contains("Missing dependencies found. Installing now"), "{}", output);
     assert!(output.contains("Compiler run successful"), "{}", output);
 });
+
+// checks that extra output works
+forgetest_init!(can_build_skip_contracts, |prj: TestProject, mut cmd: TestCommand| {
+    // explicitly set to run with 0.8.17 for consistent output
+    let config = Config { solc: Some("0.8.17".into()), ..Default::default() };
+    prj.write_config(config);
+
+    // only builds the single template contract `src/*`
+    cmd.args(["build", "--skip", "tests", "--skip", "scripts"]);
```

Comment:
nit, but should we change to `test` and `skip` to match the default folder names? or was this intentional to not match those folder names?

```suggestion
    cmd.args(["build", "--skip", "test", "--skip", "script"]);

```

Reply:
that actually makes more sense, I guess we can just do `test[s]?` and `script[s]?`

Reply:
seems reasonable to me!